File: 7-kyu/amIAfraid/langs/kata.py
def am_I_afraid(day,num):
    if day=='Monday' and num==12:
        return True
    elif day=='Tuesday' and num>95:
        return True
    elif day=='Wednesday' and num==34:
        return True
    elif day=='Thursday' and num==0:
        return True
    elif day=='Friday' and num%2==0:
        return True
    elif day=='Saturday' and num==56:
        return True
    elif day=='Sunday' and abs(num) == 666:
        return True
    else:
        return False

File: 7-kyu/amIAfraid/langs/kata.cs
using System;

public static class Kata
{
	public static bool AmIAfraid(string day, int num)
	{
		switch(day){
			case "Monday": 
			{
				if(num == 12)
				{
					return true;
				} 
				break;
			}
			
			case "Tuesday": 
			{
				if(num > 95)
				{
					return true;
				}
				break;
			}
			
			case "Wednesday": 
			{
				if(num == 34)
				{
					return true;
				}
				break;
			}
			
			case "Thursday": 
			{
				if(num == 0)
				{
					return true;
				}
				break;
			}
			
			case "Friday": 
			{
				if((num % 2) == 0)
				{
					return true;
				}
				break;
			}
			
			case "Saturday": 
			{
				if(num == 56)
				{
					return true;
				}
				break;
			}
			
			case "Sunday": 
			{
				if(Math.Abs(num) == 666)
				{
					return true;
				}
				break;
			}
		}
		return false;
	}
}

